neutrinordp/xrdp-neutrinordp.c

```diff
@@ -504,9 +518,13 @@ lxrdp_set_param(struct mod *mod, const char *name, const char *value)
     {
         settings->desktop_resize = g_text2bool(value);
     }
-    else if (g_strcmp(name, "pamusername") == 0 ||
-             g_strcmp(name, "pampassword") == 0 ||
-             g_strcmp(name, "pammsessionmng") == 0)
+    else if (g_strcmp(name, "pamusername") == 0)
+    {
+	g_strncpy(mod->pamusername, value, 255);
```

Comment:
Coding style : Use 4 spaces for indent.

Reply:
Oh, I'm sorry.
I'll follow the Coding style.

Reply:
Modified by f33a381.